spec/controllers/your_details_controller_spec.rb

```diff
@@ -100,6 +102,63 @@ module Wpcc
       end
     end
 
+    describe 'sessions' do
+      after { Timecop.return }
+
+      it "'wpcc_expires_at' key should always be updated" do
+        session[:wpcc_expires_at] = 5.minutes.ago
+        get :new, nil, some_session
+
+        expect(session[:wpcc_expires_at]).to be > 5.minutes.ago
+      end
+
+      context 'when session has expired' do
+        before do
+          wpcc_expires_at = 5.minutes.ago
+
+          Timecop.travel(Time.current + 30.minutes)
+
+          get :new,
+              nil,
+              age: 34,
+              gender: 'male',
```

Comment:
no, this is the struggle that @tomas-stefano and I had. We need to set the real session and show that it gets changed. Using some_session doesn't actually prove what we are trying to prove.